operator-pipeline-images/operatorcert/schemas/ci-schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "title": "ci.yaml validation schema",
+    "description": "json schema validating the ci.yaml",
+    "type": "object",
+    "additionaProperties": true,
+    "properties": {
+        "fbc": {
+            "description": "Config set for FBC-enabled operator bundles",
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "Whether the operator already have FBC support or not",
+                    "type": "boolean"
+                },
+                "catalog_mapping": {
```

Comment:
This section is part of a separate story. But now when you added it you can keep it.